packages/realm/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This filename would need to go into the "files" of the "package.json" too: https://github.com/realm/realm-js/blob/ec7a050c95b50672135becbb9fd8e38108b300ce/packages/realm/package.json#L52-L74